main/src/com/google/refine/grel/ControlFunctionRegistry.java

```diff
@@ -306,7 +306,8 @@ static public void registerControl(String name, Control c) {
         registerFunction("combin", new Combin());
         registerFunction("degrees", new Degrees());
         registerFunction("radians", new Radians());
-        registerFunction("randomNumber", new RandomNumber());
```

Comment:
I would not remove this registration, as it would break expressions which currently use it. We should also try to keep the function backwards compatible, meaning that arguments which were previously accepted should still be, and should be interpreted in the same way.

Reply:
Oh! Okay, makes a lot of sense. Will do 😁